Add StaticList to lab (#4194)
jpmorganchase/salt-ds: site/docs/components/static-list/index.mdx

---
title: Static list
data:

>   description: "`StaticList` manages the layout of non-interactive list items, providing a structured and composable way to display content. It is ideal for use in patterns like file upload or checkout baskets."

Reviewer: ```suggestion
  description: "`StaticList` manages the layout of non-interactive list items, providing a structured and composable way to display content. It's ideal for use in patterns like file upload or checkout baskets."
```

>   description: "The StaticList component manages the layout of non-interactive list items, providing a structured and composable way to display content. It is ideal for use in patterns like file upload or checkout baskets."

Reviewer: ```suggestion
  description: "The `StaticList` component manages the layout of non-interactive list items, providing a structured and composable way to display content. It is ideal for use in patterns like file upload or checkout baskets."
```
